starknet_py/net/gateway_client.py

```diff
@@ -143,21 +147,32 @@ async def get_state_update(
         self,
         block_hash: Optional[Union[Hash, Tag]] = None,
         block_number: Optional[Union[int, Tag]] = None,
-    ) -> BlockStateUpdate:
+        include_block: Optional[bool] = None,
```

Comment:
What's the default behaviour when sending the query?
Maybe just set it to false as default?

Reply:
I thought devnet would not like any additional arguments, but it seems that they are omitted. `return` condition would need to be changed a bit, from 
```python
        if include_block is not None:
```
to 
```python
        if res.get("block", None) is not None:
```
to allow compatibility with devnet tests.